### PR TITLE
Adjust CODEOWNERS file

### PR DESCRIPTION
**Pull request by sahaqaa**
This PR is intended to make CODEOWNERS file to be same as for https://github.com/OpenVPN/terraform-provider-cloudconnexa/blob/main/.github/CODEOWNERS

### CODE REVIEW

**Comment by sahaqaa** [comment]
I was not able to receive reply from @arslanbekov yet.
Seems that i have permissions to "bypass branch protection" and the main goal of this change is to make two CODEOWNERS files to be of similar content - adding my colleague @uddr as reviewer.

After approve i'm going to merge this PR

**Review by uddr — APPROVED** [review]

**Pull request merged**
